src/components/input.component.tsx

```diff
@@ -1,35 +1,43 @@
 import React from 'react';
-import {Text, View, TextInput, TextInputProps} from 'react-native';
-import {StyleSheet} from 'react-native';
+import {View, TextInputProps} from 'react-native';
+import styled from 'styled-components/native';
 
 type Props = TextInputProps & {
   label: string;
+  isValid: boolean;
 };
 
-export const InputComponent = ({label, ...rest}: Props) => {
+export const InputComponent = ({label, isValid, ...TextInputProps}: Props) => {
```

Comment:
```suggestion
export const InputComponent = ({label, isValid, ...textInputProps}: Props) => {
```
como o `textInputProps` é um objeto, a gente sempre nomeia com camelCase. classes, interfaces/tipos, componentes e outras _cositas_ com PascalCase 😉